advanced/creating-a-skin.md

```diff
@@ -98,7 +98,7 @@ Your skin.less file should now look like this:
 ```
 Switch to the web browser. Select your skin from the *Skin menu*. It should show a fiery red editor
 
-![**{{site.productname}} skin SDK for Silver theme**]({{site.baseurl}}/images/SDKforsilverCustomExample.png)
```

Comment:
This formats incorrectly, as you can't have bold alt text so it actually includes the asterisks verbatim.